examples/barchart.html

```diff
@@ -12,12 +12,14 @@ <h1 class="my-5">Bar chart with filters</h1>
         <bar-chart-with-filters :chart-data="chartData"
                                 :filter-config="filterConfig"
                                 :indicators="indicators"
+                                :format-function="formatFunction"
                                 v-model="selections"></bar-chart-with-filters>
     </div>
 </div>
 
 <script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.7.1/Chart.min.js"></script>
 <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
+<script src="//cdnjs.cloudflare.com/ajax/libs/numeral.js/2.0.6/numeral.min.js"></script>
```

Comment:
Have you used a cdn to prevent the vue-charts becoming too bulky? Is this standard practice on repos published on npm?

Reply:
This is just the simplest way to get the example page working, and yes, there's no need to pull numeral in as a dependency for the chart package since the chart component doesn't use it, we just want it to work in this test example.  Not sure how I lost the protocol from that link though, I'll add that in.